astropy/coordinates/sky_coordinate.py

```diff
@@ -106,6 +106,8 @@ class SkyCoord(ShapedLikeNDArray):
     argument ``representation_type='cartesian'`` (for example) along with data
     in ``x``, ``y``, and ``z``.
 
+    See also: http://docs.astropy.org/en/stable/coordinates/
```

Comment:
I find it a bit risky to hard wire links hire, there should be a way to link to this page via an intersphinx. I'll try to dig up a solution, but please ping me for it next week.

Reply:
```
:ref:`astropy-coordinates`
```

Reply:
I think this would slightly beat the point - the idea was mostly to help people who do `SkyCoord?` in ipython and want to see actual usage examples or learn how things work internally. So, a proper http link, ready to be copy & pasted, would seem better.

Reply:
And how do you make sure that the links don't broke over time? Also, this way it's not in sync with the version being used but always points to the latest release.

Reply:
Quoting from the astropy docs on writing docstrings: "A guiding principle is that human readers of the text are given precedence over contorting docstrings so our tools produce nice output."

As @mhvk said, a key driver for this is for people that are looking at docs interactively within ipython or jupyter.  

Maybe there is some technical solution to ensuring that the links are auto-generated somehow to guarantee correctness, but is it worth the effort given the million other things that need to be done?

Perfect is the enemy of good.  :smile:

Reply:
OK. So how about I add a new link checking job (I need to dig up a solution for it, but I'm sure there is a suitable solution) to the CI matrix, even if it just runs on the cron job?